swap_meet/vendor.py

```diff
@@ -1,2 +1,79 @@
+from .item import Item
+
+# wave 1
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        self.inventory = [] if inventory is None else inventory
+    
+    def add(self,item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self,item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
```

Comment:
I'd consider flipping the logic so that your 'main' logic on lines 14 and 15 are not inside a conditional if block, but in the main level of indentation. 

Then you'd be able to use a guard clause to check if an item is not in inventory and immediately return False 

```suggestion
        if item not in self.inventory:
            return False
        self.inventory.remove(item)
        return item
```